### DISCUSSION TITLE
Is It possible for fzf_lua to open file on correct line

### DISCUSSION THREAD

**Discussion by kevinrobayna** [ibhagwan/fzf-lua#1384] [Q&A] [answered]
Coming from Jetbrains and many other IDEs i've been used to when opening a file pass the line at the end with the format `:123`

Has anyone manage to do this with fzf? 

Ideally when searching like this it should show the file 
![image](https://github.com/user-attachments/assets/ae452be7-715d-4679-869b-9e1e61dacc29)
![image](https://github.com/user-attachments/assets/06125028-a66d-4574-8c2e-d753c8628278)

**Comment by ibhagwan** [comment]
Unfortunately since fzf is an external process  it’s a bit more involved to be able to transform the query and ignore anything past the `:` (so you can still have a fuzzy match), perhaps with the recent new features we might get closer to this ability but right now if possible would be a terrible hack.

Falling back to the full path written in the query prompt is possible but it would be useless as it would require the path be written fully and perfectly (i.e. no fuzzy search, no completion) and is therefore useless.

**Reply by kevinrobayna** [reply]
Yeah that makes sense!

**Comment by ibhagwan** [comment, marked as answer]
FYI, fzf 0.59 made this possible, update to https://github.com/ibhagwan/fzf-lua/commit/8cdf9aaf1c07ff5820b51ef7432ca45ab07ea982 and use (works with any file-type picker):

```lua
:FzfLua files line_query=true
```

Once `:<line>` is detected it will be parsed and applied to both the previewer and action (enter, etc):
![image](https://github.com/user-attachments/assets/281a309f-8d82-4a73-9449-3ee0675d5dfc)

**Reply by kevinrobayna** [reply]
Oh this is awesome!!!
